scripts/re_matrix.py

```diff
@@ -2,17 +2,45 @@
 import os
 import md5mod
 import urllib.request
-import read_argfile
+import requests
+
+def read_key_value_file(filename="argfile.conf"):
+    data = {}
+    with open(filename, "r") as file:
+        for line in file:
+            line = line.strip()
+            if line and "=" in line:
+                key, value = line.split("=", 1)
+                data[key.strip()] = value.strip()
+    return data
+
+def download_file(url, save_path):
+    try:
+        urllib.request.urlretrieve(url, save_path)
+        return save_path
+    except Exception as e:
+        return f"Error download file: {e}"
+
+def fetch_dtk_tags():
+    dtk_registry_api = "https://quay.io/v1/repositories/build-and-sign/pa-driver-toolkit/tags"
+    #  Get DTK tags
+    response = requests.get(dtk_registry_api)
+    kernel_json = (response.json())
+    kernel_list = "data/kernel-list.json"
```

Comment:
we have this filename hardcoded here AND hardcoded further down in the file so if we want to change it we have to do it in (at least) two places and if we miss one it will cause hard to debug issues, so we probably want it defined in only one place, so we cuold set it as a global variable, or pass it into this function as a parameter, or something else?  (or see the next comment :) )